base/code/src/containers/About.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { getApiData } from '../services/api';
+
+const About: React.FC<{}> = () => {
+  const [title, setTitle] = React.useState('');
+  const [content, setContent] = React.useState('');
```

Comment:
You should also handle the loading and error states. That's quite simple with the `useAsync` hook from `react-use`